### PR TITLE
Add request specs WIP

### PR DESCRIPTION
**Pull request by fancydaniel**
Added request specs. Deleted other automatically generated tests. 
5 Failures still to fix.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -23,6 +23,7 @@ end
 
 group :test do
   gem 'database_cleaner'
+  gem 'rails-controller-testing'
```
Comment (jendiamond) [inline Gemfile:26]: why did you add this gem?

Comment (fancydaniel) [inline Gemfile:26]: Deprecation warning during testing required this gem.
